modules/user/user.graphql.js

```diff
@@ -49,6 +49,7 @@ input UserInput {
 }`;
 const userUpdateInput = `
 input UserUpdateInput {
+    _id: ID
```

Comment:
Should it be required?

Reply:
Nope, it is just for the convenience, instead of trimming "_id" when sending data, it is easier to just mention that user input might contain "_id" field. "_id" cannot be updated anyway.